staging/src/k8s.io/mount-utils/mount_helper_unix.go

```diff
@@ -199,3 +201,49 @@ func PathExists(path string) (bool, error) {
 	}
 	return false, err
 }
+
+var (
+	miBug     bool
```

Comment:
Can you consider adding a comment that this variable is meant only for `kernelHasMountinfoBug`

Reply:
Or maybe rename the variable? `miBug` does not say much, `stableMountRead`? `kernelMountCursor`? Same for `kernelHasMountinfoBug` -> `kernelHasStableMountRead`?

Reply:
Well, these two variables are indeed only used by `kernelHasMountinfoBug`. Since Golang lacks `static` qualifier, they have to be global. Sure, it makes sense to add a comment telling what these are for.

I am using short names here because the scope is pretty small, based on recommendations in https://google.github.io/styleguide/go/decisions#variable-names.

As for `kernelHasMountinfoBug` -> `kernelHasStableMountRead`, I don't want to negate the return value, to emphasize we're using a kludge to work around the bug. Something that is named like `kernelHasStableMountRead` would suggest it is a feature, while in fact it's not, it's just the bug was fixed.

Reply:
Renamed; added a comment.